barman/cli.py

```diff
@@ -484,6 +484,15 @@ def backup_completer(prefix, parsed_args, **kwargs):
             default=None,
             type=check_non_negative,
         ),
+        argument(
+            "--keepalive-interval",
+            help="An interval, in seconds, at which a heartbeat query will be sent "
+            "to the server to keep the connection alive during an Rsync backup. "
+            "Default is 60 seconds. A value of 0 disables it",
```

Comment:
```suggestion
```

I think we can keep the code as it is and remove the information about the default.
We should add the new parameter to the docs, and there we can explain the default value for the setting, and what it does in more details.